tests/test-sources/modules/filetypes.nix

```
@@ -38,4 +38,21 @@
       };
     };
   };
+
+  default-empty.module =
+    { config, ... }:
+    {
+      files.test = { };
+
+      assertions = [
+        {
+          assertion = builtins.match ".*vim\.filetype\..*" config.content == null;
```

BTW you can use [`lib.hasInfix`](https://noogle.dev/f/lib/hasInfix) when you don't actually need a regex:

```suggestion
          assertion = lib.hasInfix "vim.filetype." config.content;
```

---

Right. I already found out about it yesterday when working on modules/output refactoring. I intended to change it in other PRs, but looks like I forgot about this.